src/haddock/clis/cli_clean.py

```diff
@@ -9,19 +9,22 @@
 deleted.
 
 The <run_directory> can either be a whole HADDOCK3 run folder or a
-specific folder of the workflow step.
+specific folder of the workflow step. <ncores> defined the number of
+threads to use.
 
 Usage::
 
     haddock3-clean -h
     haddock3-clean <run_directory>
     haddock3-clean run1
     haddock3-clean run1/1_rigidbody
+    haddock3-clean run1 -n  # uses all cores
```

Comment:
I assume that without the -n argument all cores are used as well, correct?

Reply:
the `-n` alone will use all cores. withOUT `-n` only 1 will be used.